task30/src/test/java/by/tut/tests/MultiselectTest.java

```diff
@@ -0,0 +1,59 @@
+package by.tut.tests;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import static org.testng.Assert.assertEquals;
+
+@Test(groups = "task40")                                                                                                               //POINT 5
+public class MultiselectTest {
```

Comment:
Please re-write this point using Select class of WebDriver api

Reply:
fixed

I've used HashSet to get random items that are not repeated.